.github/workflows/release.yaml

```diff
@@ -44,11 +46,18 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Build and Push
+      - name: Get version
+        id: get-version
+        env:
+          ENVBUILDER_RELEASE: "t"
```

Comment:
review: the updated script will emit `-dev` versions unless this is explicitly set. In this case it will validate that the version is an annotated tag.